### PR TITLE
📖 Add docs for API

### PR DESCRIPTION
**Pull request by azeemshaikh38**
PR adds docs to show REST API usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ Then click "Add More Scanning Tools."
 
 The workflow is preconfigured to run on every repository contribution. After making a code change, you can view the results for the change either through the Scorecard Badge, Code Scanning Alerts or GitHub Workflow Runs.
 
+### REST API
+Starting with scorecard-action:v2, users can use a REST API to query their latest run results. This requires setting `publish_results: true` for the action and enabling `id-token: write` permission for the job (needed to access GitHub OIDC token). The API is available here: https://api.securityscorecards.dev.
```
Comment (naveensrinivasan) [inline README.md:114]: How about including a link to an example like this for `publish_results` https://github.com/ossf/scorecard/blob/d13ba3f3355b958d5d62edc47282a2e7ed9fa7c1/.github/workflows/scorecard-analysis.yml#L39 and for `id-token` https://github.com/ossf/scorecard/blob/d13ba3f3355b958d5d62edc47282a2e7ed9fa7c1/.github/workflows/scorecard-analysis.yml#L22?

This makes it clear for anyone new.

**Review by naveensrinivasan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (azeemshaikh38) [reply README.md]: Done

**Review by laurentsimon — APPROVED** [review]
Nice! So I think we're ready to move forward with the dependency visualization on pull requests?

**Comment by azeemshaikh38** [comment]
> Nice! So I think we're ready to move forward with the dependency visualization on pull requests?

I think so. Maybe after the launch is completed, but yes.

**Pull request merged**